content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -226,6 +226,57 @@ As a client, you can request `BOOKMARK` events by setting the
 assume bookmarks are returned at any specific interval, nor can clients assume that
 the API server will send any `BOOKMARK` event even when requested.
 
+## Streaming lists
+
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+On large clusters, retrieving the collection of some resource types may result in
+a significant increase of resource usage (primarily RAM) on the control plane.
+In order to alleviate its impact and simplify the user experience of the **list+watch**
+pattern, we added a support for requesting the initial state (previously requested via
```

Comment:
```suggestion
pattern, Kubernetes - including version {{< skew currentVersion >}}- provides alpha support for requesting the
initial state (which you would otherwise request separately via
```

Howe about